dotCMS/src/main/java/com/dotmarketing/util/ConvertToPOJOUtil.java

```diff
@@ -0,0 +1,385 @@
+     * @param results
+     * @return
+     */
+    public static List<Identifier> convertDotConnectMapToIdentifier(final List<Map<String,String>> results)
+            throws ParseException {
+        final List<Identifier> ret = new ArrayList<>();
+        if(results == null || results.isEmpty()){
+            return ret;
+        }
+
+        for (Map<String, String> map : results) {
+            final Identifier i = new Identifier();
+            i.setAssetName(map.get("asset_name"));
+            i.setAssetType(map.get("asset_type"));
+            i.setHostId(map.get("host_inode"));
+            i.setId(map.get("id"));
+            i.setParentPath(map.get("parent_path"));
+
+            if (map.get(SYSPUBLISH_DATE) != null && !map.get(SYSPUBLISH_DATE).isEmpty()){
+                i.setSysPublishDate(df.parse(map.get(SYSPUBLISH_DATE) ));
+            }
+
+            if (map.get(SYSEXPIRE_DATE) != null && !map.get(SYSEXPIRE_DATE).isEmpty()){
+                i.setSysExpireDate(df.parse(map.get(SYSEXPIRE_DATE) ));
+            }
+
+            ret.add(i);
+        }
+        return ret;
+    }
+
+    public static List<Folder> convertDotConnectMapToFolder(final List<Map<String,String>> results)
+            throws ParseException {
+
+        Folder folder;
+        List<Folder> ret;
+
+        ret = new ArrayList<>();
+
+        if(results != null && !results.isEmpty()){
+            for (Map<String, String> map : results) {
+                folder = new Folder();
+                folder.setInode(map.get(INODE));
+                folder.setName(map.get("name"));
+                folder.setTitle(map.get(TITLE));
+
+                if (map.get(SHOW_ON_MENU) != null && !map.get(SHOW_ON_MENU).isEmpty()) {
+                    folder.setShowOnMenu(Boolean.parseBoolean(map.get(SHOW_ON_MENU)));
+                }
+
+                if (map.get(SORT_ORDER) != null && !map.get(SORT_ORDER).isEmpty()){
+                    folder.setSortOrder(Integer.parseInt(map.get(SORT_ORDER)));
+                }
+
+                folder.setFilesMasks(map.get("files_masks"));
+
+                folder.setIdentifier(map.get(IDENTIFIER));
+
+                folder.setDefaultFileType(map.get("default_file_type"));
+
+                if (map.get(MOD_DATE) != null && !map.get(MOD_DATE).isEmpty()){
+                    folder.setModDate(df.parse(map.get(MOD_DATE)));
+                }
+
+                ret.add(folder);
+            }
+        }
+        return ret;
+    }
+
+    public static List<Container> convertDotConnectMapToContainer(final List<Map<String,String>> results)
+            throws ParseException {
+        Container container;
+        List<Container> ret;
+
+        ret = new ArrayList<>();
+
+        if(results != null && !results.isEmpty()){
+            for (Map<String, String> map : results) {
+                container = new Container();
+                container.setInode(map.get(INODE));
+                container.setCode(map.get("code"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "setCode"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)